.github/workflows/exhaustive.yml

```diff
@@ -1605,45 +3221,46 @@ jobs:
 
       - name: Convert ZLib
         run: |
-          cd ${{env.benchmark_conv_dir}}/no-alltypes__only_l_sol/zlib-1.2.11
+          cd ${{env.benchmark_conv_dir}}/expand_macros_no_alltypes__only_l_sol/zlib-1.2.11
           ${{env.port_tools}}/convert_project.py \
             --skip '/.*/test/.*' \
             --includeDir ${{env.include_dir}} \
             --prog_name ${{env.builddir}}/bin/3c \
+            --expand_macros_before_conversion \
             --extra-3c-arg=-only-l-sol \
             --project_path . \
             --build_dir build
 
       - name: Copy 3c stats of ZLib
         run: |
-          cd ${{env.benchmark_conv_dir}}/no-alltypes__only_l_sol/zlib-1.2.11
+          cd ${{env.benchmark_conv_dir}}/expand_macros_no_alltypes__only_l_sol/zlib-1.2.11
           mkdir 3c_performance_stats/
           cp *.json 3c_performance_stats/
 
       - name: Upload 3c stats of ZLib
         uses: actions/upload-artifact@v2
         with: 
           name: ZLib__only_l_sol_noalltypes
```

Comment:
We should add `no_macro` to the artifact name. I.e., Change the generate-workflow.py to add this.

Reply:
I'm a bit confused by the particular diff you commented on; the GitHub review UI seems to be working in a way I don't understand.  But I added `no_expand_macros` to the "subvariant" name (which is used in the job name as well as the artifact name) when macro expansion is off for consistency with `expand_macros` when it is on.  Was that what you were asking for?

These names are getting rather long and unwieldy.  If you have a simple idea to improve them, I can try it in this PR, otherwise we can keep the names for now and change them later if we want.

Reply:
Yes, those are the changes I was asking for. 
The changes look good. 
We can keep the long names for now and change them later.